types/src/front/lib/api/credentials.ts

```diff
@@ -55,10 +59,11 @@ export const credentialsFromProviders = (
 
 export const dustManagedCredentials = (): CredentialsType => {
   return {
-    OPENAI_API_KEY: DUST_MANAGED_OPENAI_API_KEY,
     ANTHROPIC_API_KEY: DUST_MANAGED_ANTHROPIC_API_KEY,
-    TEXTSYNTH_API_KEY: DUST_MANAGED_TEXTSYNTH_API_KEY,
     AZURE_OPENAI_API_KEY: DUST_MANAGED_AZURE_OPENAI_API_KEY,
     AZURE_OPENAI_ENDPOINT: DUST_MANAGED_AZURE_OPENAI_ENDPOINT,
+    MISTRAL_AI_API_KEY: DUST_MANAGED_MISTRAL_AI_API_KEY,
```

Comment:
This as well is for the release of Mistral as an assistant and can be kept in the next PR :+1: